moveit_planners/ompl/ompl_interface/test/test_planning_context_manager.test

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<launch>
+    <test pkg="moveit_planners_ompl" type="test_planning_context_manager" test-name="test_planning_context_manager" />
+</launch>
```

Comment:
There is no ROS environment loaded here. Do you need a rostest with a launch file after all? 
Can the test run as a plain gtest as well?

Reply:
I need a node handle to call `getPlanningRequest`, but the node handle is never used. As mentioned in the pr description:
>Ideally, this test could be completely ROS independent if I had a way to create a "dummy" ros::NodeHandle without actually starting a roscore. The only method that needs this ROS handle is loadConstraintApproximations, which I'm not testing here.

I wish I could come up with a better solution. It is unfortunate that a ROS dependency got introduced in a non-ROS specific part of MoveIt. Technically, the code for the constraint approximations could be refactored to avoid this ros node handle dependency, but that would introduce breaking API changes for users, I think.